doc/google-assistant.md

```diff
@@ -0,0 +1,62 @@
+# Google Assistant
+
+You can use Google Assistant to access Thingpedia devices!
+
+First, follow the instructions [here](https://developers.google.com/assistant/actions/actions-sdk) to create an Action package for Google Assistant.
+
+Specifically, the Action package JSON file should be replaced with the following, taking care to replace `<YourActionName>` with your Action name.
+
+```
+{
+  "actions": [
+    {
+      "description": "Default Welcome Intent",
+      "name": "MAIN",
+      "fulfillment": {
+        "conversationName": "<YourActionName>"
+      },
+      "intent": {
+        "name": "actions.intent.MAIN",
+        "trigger": {
+          "queryPatterns": [
+            "talk to <YourActionName>"
+          ]
+        }
+      }
+    },
+    {
+      "name": "TEXT",
+      "intent": {
+        "name": "actions.intent.TEXT",
+        "trigger": {
+          "queryPatterns": [
+            "bye",
+            "goodbye"
+          ]
+        }
+      },
+      "fulfillment": {
+        "conversationName": "<YourActionName>"
+      }
+    }
+  ],
+  "conversations": {
+    "<YourActionName>": {
+      "name": "<YourActionName>",
+      "url": "https://almond.stanford.edu/me/api/gassistant/fulfillment"
```

Comment:
If I do this, I get full Almond, approved-only. Correct?
If I am a Thingpedia developer, how do I test my own devices in Google Assistant?

Reply:
Fixed authentication